### PR TITLE
[WIP] Improve the documentation

### PR DESCRIPTION
**Pull request by nikever**
Not really sure how to structure the documentation for the tool. I added a new possible skeleton.

Any feedbacks?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,123 @@
-# Furyagent v0.2.3
+<h1 align="center">
+  <img src="docs/assets/furyagent.png" width="200px"/><br/>
+  Furyagent
+</h1>
+
+<p align="center">Automate some of the Kubernetes cluster management burocracy</p>
```
Comment (ralgozino) [inline README.md:6]: I'd drop "some" from this sentence

File: README.md [hunk 2]
```diff
@@ -1,61 +1,123 @@
...
 [![Build Status](http://ci.sighup.io/api/badges/sighupio/furyagent/status.svg?ref=refs/tags/v0.2.3)](http://ci.sighup.io/sighupio/furyagent)
+![Release](https://img.shields.io/github/v/release/sighupio/furyagent?label=Release)
+![License](https://img.shields.io/github/license/sighupio/furyagent)
+[![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack&label=Slack)](https://kubernetes.slack.com/archives/C0154HYTAQH)
 
-## Install
+<!-- <KFD-DOCS> -->
 
-You can find `furyagent` binaries on the [Releases page](https://github.com/sighupio/furyagent/releases).
+**Furyagent** is a simple CLI tool to automate some common tasks in a Kubernetes cluster management.
```
Comment (ralgozino) [inline README.md:15]: same as above

File: README.md [hunk 3]
```diff
@@ -1,61 +1,123 @@
...
 
-Supported architectures are (64 bit):
+Furyagent can help you to:
 
--   `linux`
--   `darwin`
+1. Backup and restore etcd
+2. Dispatch etcd and master certificates
```
Comment (ralgozino) [inline README.md:20]: what if we say instead  "etcd and Kubernetes PKI creation and distribution"?

File: README.md [hunk 4]
```diff
@@ -1,61 +1,123 @@
...
 
-Download right binary for your architecture and add it to your PATH. Assuming it's downloaded in your
-`~/Downloads` folder, you can run following commands (replacing `{arch}` with your architecture):
+## Installation
 
-```shell
-chmod +x  ~/Downloads/furyagent-{arch}-amd64 && mv ~/Downloads/furyagent-{arch}-amd64 /usr/local/bin/furyagent
+You can find `furyagent` binaries on the [Releases page](https://github.com/sighupio/furyagent/releases).
+
+To download the latest release, run:
```
Comment (ralgozino) [inline README.md:28]: To download the latest release directly from the command line, run:

File: README.md [hunk 5]
```diff
@@ -1,61 +1,123 @@
...
-```
+> Remeber to add `openvpn` to your `$PATH` by adding to your `.bashrc` or `.zshrc`:
+>
+> `export PATH=$(brew --prefix openvpn)/sbin:$PATH`
 
-in your .zshrc or .bashrc .
+## Storage
 
-## Usage
+Furyagent uses an utility S3 Bucket to easily bookkeep all the information and dispatch certificates easily.
```
Comment (ralgozino) [inline README.md:68]: furyagent also supports the `local` provider with a `path` where you'll find all the certs

File: README.md [hunk 6]
```diff
@@ -69,84 +131,50 @@ furyagent
...
-## Workflow
+## Configure Furyagent
 
-1. Write a [`furyagent.yml`](furyagent.yml)
-2. Generate certificates (by hand at the moment)
-3. `furyagent init -d /path/to/cert/dir --config /path/to/furyagent.yml [etcd|master]` to upload certificates
-4. Then on the nodes: `furyagent configure --config /path/to/furyagent.yml [etcd|master]` to download the certificates to the correct directory specified in the config file
-5. if needed: to backup the state of etcd through `furyagent backup --config /path/to/furyagent.yml etcd`
-6. if needed: to restore the state of etcd, stop etcd, run `furyagent restore --config /path/to/furyagent.yml etcd`, restart etcd
+`furyagent` uses an utility S3 Bucket. To configure access, `furyagent` reads a `Furyagent.yml` config file.
```
Comment (ralgozino) [inline README.md:141]: I think it has to be all under case: `furyagent.yml`

File: README.md [hunk 7]
```diff
@@ -69,84 +131,50 @@ furyagent
...
+  provider: s3  
+  url: "http://s3.eu-central-1.amazonaws.com"
+  aws_access_key: <AWS_ACCESS_KEY>
+  aws_secret_key: <AWS_SECRET_KEY>
+  bucketName: <AWS_BUCKET_NAME>
+  region: <AWS_BUCKET_REGION>
+```
 
-We still use `go mod` as golang package manager. Once you have that installed you can run `go mod vendor` and `go build` or `go install` should run without problems
+The `storage` section contains the AWS credentials and the location of the S3 bucket. Make sure that the AWS service account has read/write permissions to the bucket.
```
Comment (ralgozino) [inline README.md:153]: "... and to store this file encrypted or in a secure way" ?

File: README.md [hunk 8]
```diff
@@ -69,84 +131,50 @@ furyagent
...
 
-## Storage
+## Tasks
```
Comment (ralgozino) [inline README.md:155]: I'd drop this title and leave all under Usage

File: README.md [hunk 9]
```diff
@@ -69,84 +131,50 @@ furyagent
...
 
-There is going to be one and only one bucket per cluster.
+> :warning: Make sure you have configured
```
Comment (ralgozino) [inline README.md:157]: I think there's something missing here

File: README.md [hunk 10]
```diff
@@ -1,61 +1,123 @@
...
 
-```
-export PATH=$(brew --prefix openvpn)/sbin:$PATH
-```
+> Remeber to add `openvpn` to your `$PATH` by adding to your `.bashrc` or `.zshrc`:
+>
+> `export PATH=$(brew --prefix openvpn)/sbin:$PATH`
 
-in your .zshrc or .bashrc .
+## Storage
```
Comment (ralgozino) [inline README.md:66]: I would move this section inside "Configuration"

File: README.md [hunk 11]
```diff
@@ -69,84 +131,50 @@ furyagent
...
-│   └── master
+│   └── etcd
 └── restore
-    ├── etcd
-    └── master
+    └── etcd
 ```
 
-## Workflow
+## Configure Furyagent
```
Comment (ralgozino) [inline README.md:139]: just "Configuration"

Comment (ralgozino) [reply README.md:139]: move this before Usage

File: README.md (hunk 1 above)
Comment (nikever) [reply README.md:6]: ✅

File: README.md (hunk 2 above)
Comment (nikever) [reply README.md:15]: ✅

File: README.md (hunk 3 above)
Comment (nikever) [reply README.md:20]: ✅

File: README.md (hunk 4 above)
Comment (nikever) [reply README.md:28]: ✅

File: README.md (hunk 5 above)
Comment (nikever) [reply README.md:68]: image

File: README.md (hunk 7 above)
Comment (nikever) [reply README.md:153]: ✅

File: README.md (hunk 8 above)
Comment (nikever) [reply README.md:155]: ✅

File: README.md (hunk 10 above)
Comment (nikever) [reply README.md:66]: ✅

File: README.md (hunk 11 above)
Comment (nikever) [reply README.md:139]: ✅

File: README.md (hunk 9 above)
Comment (nikever) [reply README.md:157]: ✅

File: README.md (hunk 6 above)
Comment (ralgozino) [reply README.md]: default is all lower case: https://github.com/sighupio/furyagent/blob/master/cmd/root.go#L62 @nikever

File: README.md [hunk 12]
```diff
@@ -179,7 +182,7 @@ If you need to revoke access to any user, you can do it with the following comma
 furyagent config openvpn-client --client-name foo --revoke --config /etc/fury/furyagent.yml
 ```
 
-### SSH management
+## SSH management
```
Comment (ralgozino) [inline README.md:185]: Shouldn't this be uder Usage? i.e. e ### instead of ## (as it was before)?

**Pull request merged**